File: PlatziCodingChallenge_Dia31/TeoremaPitagoras.py
# Teorema de Pitágoras en Python

# Función que calcula la distancia entre dos puntos
def distance(pointA, pointB):
    return ((pointB.x - pointA.x)**2 + (pointB.y - pointA.y)**2)**.5

# Clase para crear puntos
class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y

# Datos
puntoA = Point(3, 4)
puntoB = Point(5, 7)

# Ejecución
print(round(distance(puntoA, puntoB), 6)) # 3.605551

File: Dia31/TeoremaPitagoras.js
// Teorema de Pitágoras en JavaScript

// Función que calcula la distancia entre dos puntos
function distance(pointA, pointB) {
    return Math.sqrt(Math.pow((pointB.x - pointA.x), 2) + Math.pow((pointB.y - pointA.y), 2))
}

// Datos
const puntoA = {
    x: 3,
    y: 4
}
const puntoB = {
    x: 5,
    y: 7
}

// Ejecutando la función
console.log(distance(puntoA, puntoB).toFixed(6)) // 3.605551